code/game/objects/items/weapons/tanks/jetpack.dm

```diff
@@ -27,7 +27,6 @@
 	. = ..()
 	if(air_contents.total_moles < 5)
 		user << "<span class='danger'>The meter on \the [src] indicates you are almost out of gas!</span>"
-		playsound(user, 'sound/effects/alert.ogg', 50, 1)

```

Comment:
i didnt realise it was in the examine proc. In whcih case i agree with this solution, sounds shouldn't be playing in examine.

I thought this was an automated warning playing from a process stack. my comment was in that belief